### PR TITLE
buff: weapon buffs in a separate frame, configure rowsize for debuffs/weapon buffs

### PR DESCRIPTION
**Pull request by dein0s**
Subj

### CODE REVIEW

File: api/config.lua [hunk 1]
```diff
@@ -54,7 +54,9 @@ function pfUI:LoadConfig()
   pfUI:UpdateConfig("buffs",      nil,           "weapons",          "1")
   pfUI:UpdateConfig("buffs",      nil,           "size",             "24")
   pfUI:UpdateConfig("buffs",      nil,           "spacing",          "5")
-  pfUI:UpdateConfig("buffs",      nil,           "rowsize",          "16")
+  pfUI:UpdateConfig("buffs",      nil,           "wepbuffrowsize",  "2")
+  pfUI:UpdateConfig("buffs",      nil,           "buffrowsize",     "16")
+  pfUI:UpdateConfig("buffs",      nil,           "debuffrowsize",   "16")
```
Comment (shagu) [inline api/config.lua]: :(

File: modules/buff.lua [hunk 2]
```diff
@@ -201,9 +225,17 @@ pfUI:RegisterModule("buff", function ()
...
+  pfUI.buff.wepbuffs.count = 0
+  pfUI.buff.wepbuffs.buttons = {}
+  for i=1,2 do
+    pfUI.buff.wepbuffs.buttons[i] = CreateBuffButton(i, "HELPFUL", 1)
+  end
+
   -- Buff Frame
   pfUI.buff.buffs = CreateFrame("Frame", "pfBuffFrame", UIParent)
-  pfUI.buff.buffs.offset = 0
+  -- pfUI.buff.buffs.offset = 0
```
Comment (shagu) [inline modules/buff.lua]: feel free to remove it if it isn't needed anymore

File: api/config.lua (hunk 1 above)
Comment (dein0s) [reply api/config.lua]: https://github.com/shagu/pfUI/issues/867 :)
or did i misunderstood your reaction?

Comment (shagu) [reply api/config.lua]: It's the indentation, there's whitespace missing infront of the numbers. :)

**Comment by dein0s** [comment]
Alright, added some conditional magic :)
Tested with default and personal configurations - everything seems fine.
Let me know if any additional changes needed.

**Comment by shagu** [comment]
Could you please rebase your PR? Thanks

**Comment by dein0s** [comment]
Done

**Pull request merged**